.github/workflows/playwright.yml

```
@@ -0,0 +1,36 @@
+---
+name: Playwright Tests
+on:
+  deployment_status:
+    paths:
+      - 'playwright/**'
+      - '.github/workflows/playwright.yml'
+defaults:
+  run:
+    working-directory: ./playwright
+env:
+  PW_XFD_URL: ${{ vars.PW_XFD_URL }}
+  PW_XFD_USERNAME: ${{ secrets.PW_XFD_USERNAME }}
+  PW_XFD_PASSWORD: ${{ secrets.PW_XFD_PASSWORD }}
+  PW_XFD_2FA_ISSUER: ${{ secrets._PW_XFD_2FA_ISSUER }}
+  PW_XFD_2FA_SECRET: ${{ secrets.PW_XFD_2FA_SECRET }}
+  PW_XFD_USER_ROLE: ${{ vars.PW_XFD_USER_ROLE }}
```

Sort environment variables alphabetically:
```suggestion
  PW_XFD_2FA_ISSUER: ${{ secrets._PW_XFD_2FA_ISSUER }}
  PW_XFD_2FA_SECRET: ${{ secrets.PW_XFD_2FA_SECRET }}
  PW_XFD_PASSWORD: ${{ secrets.PW_XFD_PASSWORD }}
  PW_XFD_URL: ${{ vars.PW_XFD_URL }}
  PW_XFD_USER_ROLE: ${{ vars.PW_XFD_USER_ROLE }}
  PW_XFD_USERNAME: ${{ secrets.PW_XFD_USERNAME }}
```